test/onevariable.jl

```diff
@@ -480,6 +480,20 @@ eeuler = Base.MathConstants.e
     for i in 0:2
         @test a[i]*(180/pi) == b[i]
     end
+
```

Comment:
While this test is quite ok, I think you should test directly the actual constructors introduced, e.g., `Taylor{Float64}(true)` and others alike.

Reply:
I've already added some direct tests for the constructors, such as `@test Taylor1{Float64}(true) == Taylor1([1.0])`, etc.; are those kinds of tests the ones you mean, or are there some others that you would suggest to add?

Reply:
Thanks!